### PR TITLE
Update encrypt data

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Work for #33477 task-7
/assign @tengqm
PTAL, thanks.

### CODE REVIEW

**Review by sftim — CHANGES_REQUESTED** [review]
Thank you. Please omit the changes to the English localization here, @fenggw-fnst 

It will be easiest to review this if the PR only touches **ONE** localization.

**Comment by fenggw-fnst** [comment]
> Thank you. Please omit the changes to the English localization here, @fenggw-fnst
> 
> It will be easiest to review this if the PR only touches **ONE** localization.

Thanks for the review, done.

File: content/zh/docs/tasks/administer-cluster/encrypt-data.md [hunk 1]
```diff
@@ -290,66 +322,73 @@ For larger clusters, you may wish to subdivide the secrets by namespace or scrip
...
 ## Decrypting all data
 
-To disable encryption at rest place the `identity` provider as the first entry in the config:
+To disable encryption at rest, place the `identity` provider as the first entry in the config
+and restart all `kube-apiserver` processes.
 -->
 ## 解密所有数据
 
-要禁用 rest 加密，请将 `identity` provider 作为配置中的第一个条目：
+要禁用 rest 加密，请将 `identity` provider 作为配置中的第一个条目
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/encrypt-data.md:362]:
```suggestion
要禁用静态加密，请将 `identity` provider 作为配置中的第一个条目
```

**Comment by fenggw-fnst** [comment]
Thanks for the review, I have updated the PR as suggested.

File: content/zh/docs/tasks/administer-cluster/encrypt-data.md [hunk 2]
```diff
@@ -290,66 +322,73 @@ For larger clusters, you may wish to subdivide the secrets by namespace or scrip
...
-2. 重新启动所有 `kube-apiserver` 进程以确保每台服务器都可以使用新密钥进行解密
-3. 将新密钥设置为 `keys` 数组中的第一个条目，以便在配置中使用其进行加密
-4. 重新启动所有 `kube-apiserver` 进程以确保每个服务器现在都使用新密钥进行加密
-5. 运行 `kubectl get secrets --all-namespaces -o json | kubectl replace -f -` 以用新密钥加密所有现有的秘密
-6. 在使用新密钥备份 etcd 后，从配置中删除旧的解密密钥并更新所有密钥
+1. 重新启动所有 `kube-apiserver` 进程以确保每台服务器都可以使用新密钥进行解密
+1. 将新密钥设置为 `keys` 数组中的第一个条目，以便在配置中使用其进行加密
+1. 重新启动所有 `kube-apiserver` 进程以确保每个服务器现在都使用新密钥进行加密
+1. 运行 `kubectl get secrets --all-namespaces -o json | kubectl replace -f -`
+   以用新密钥加密所有现有的秘密
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/encrypt-data.md:349]:
```suggestion
   以用新密钥加密所有现有的 Secret
```

File: content/zh/docs/tasks/administer-cluster/encrypt-data.md [hunk 3]
```diff
@@ -290,66 +322,73 @@ For larger clusters, you may wish to subdivide the secrets by namespace or scrip
...
 
-To disable encryption at rest place the `identity` provider as the first entry in the config:
+To disable encryption at rest, place the `identity` provider as the first entry in the config
+and restart all `kube-apiserver` processes.
 -->
 ## 解密所有数据
 
-要禁用 rest 加密，请将 `identity` provider 作为配置中的第一个条目：
+要禁用静态加密，请将 `identity` provider 作为配置中的第一个条目
+并重新启动所有 `kube-apiserver` 进程。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/encrypt-data.md:363]:
```suggestion
要禁用静态加密，请将 `identity` provider
作为配置中的第一个条目并重新启动所有 `kube-apiserver` 进程。
```

**Comment by fenggw-fnst** [comment]
Done, thanks.

**Pull request merged**